build.rs

```diff
@@ -20,8 +20,22 @@ fn main() {
     env::var("PROFILE").unwrap()
   };
 
+  // Equivilent to target arch != host arch
+  let is_cross_compile = env::var("CARGO_CFG_TARGET_ARCH").unwrap().as_str()
```

Comment:
This might not have the intended effect. Some targets might vary from host, but will still be compatible with the host. I.E. `x86_64-unknown-linux-musl`

Reply:
`is_cross_compile` might be a bad name for this one.